deployment/modules/gcp/gcs/main.tf

```
@@ -67,10 +67,27 @@ resource "google_spanner_database" "log_db" {
   deletion_protection = !var.ephemeral
 }
 
+resource "google_spanner_database" "log_antispam_db" {
+  count    = var.create_antispam ? 1 : 0
+  instance = google_spanner_instance.log_spanner.name
+  name     = "${var.base_name}-db_dedup"
```

Should the `dedup` in the database name be replaced with `antispam`? I find it a bit weird to have both `-` and `_` in the name.

```suggestion
  name     = "${var.base_name}-antispam-db"
```

---

Yeah, you're not wrong, let me see how much breakage that's going to cause :)